### PR TITLE
feat(demo): realistic error rate

### PR DESCRIPTION
**Pull request by scefali**
Previously, the error rates for scenario 1 in the demo were 0% and 100%:

Screen Shot 2021-04-05 at 3 30 00 PM

This is both unrealistic and wrong (the React transaction should be a failure). To fix this, I added a scenario 1b which is similar to scenario 1 but does _not_ produce any errors and also fixed the React transaction related to the error. This gives me a more realistic error rate:

Screen Shot 2021-04-05 at 3 28 11 PM

Also made some other minor changes to event JSON.

### CODE REVIEW

File: src/sentry/demo/data/scen1b/python_transaction.json [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        "tags": {"status": "ok"},
+        "same_process_as_parent": true
+      }
+    ],
+    "start_timestamp": 1615837032.402719,
+    "timestamp": 1615837034.460185,
+    "title": "checkout",
+    "transaction": "checkout",
+    "type": "transaction",
+    "user": {"email": "zllm@yahoo.com"},
```
Comment (wmak) [inline src/sentry/demo/data/scen1b/python_transaction.json:163]: just noticed these have `@yahoo.com` and while we generate users later, should we change 'em to `blah@example.com` or something just to be on the safe side?

File: src/sentry/demo/data/scen1b/python_transaction.json [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    "datetime": "2021-03-15T19:37:14.460185+00:00",
+    "tags": [
+      ["browser", "Safari 12.1.1"],
+      ["browser.name", "Safari"],
+      ["client_os", "Mac OS X 10.13.6"],
+      ["client_os.name", "Mac OS X"],
+      ["device", "Mac"],
+      ["device.family", "Mac"],
+      ["environment", "prod"],
+      ["http.status_code", "500"],
```
Comment (wmak) [inline src/sentry/demo/data/scen1b/python_transaction.json:17]: Probably should change this to 200 since this transaction isn't erroring anymore

**Review by wmak — APPROVED** [review]

File: src/sentry/demo/data/scen1b/python_transaction.json (hunk 1 above)
Comment (scefali) [reply src/sentry/demo/data/scen1b/python_transaction.json:163]: @wmak these emails were auto-generated to begin with but I can just remove them in the template (we end up replacing it later anyways).

File: src/sentry/demo/data/scen1b/python_transaction.json (hunk 2 above)
Comment (scefali) [reply src/sentry/demo/data/scen1b/python_transaction.json:17]: @wmak good catch!

**Pull request merged**